docs/source/examples/predict_bike_trips.ipynb

```diff
@@ -0,0 +1,414 @@
+    "    time_index='starttime',\n",
+    "    window_size='13h',\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Finding the Training Examples\n",
+    "\n",
+    "Now, let's run a search to get the training examples by using the following parameters:\n",
+    "\n",
+    "* The trips sorted by the start time.\n",
+    "* `num_examples_per_instance` to find the number of training examples per station. In this case, the search will return all existing examples.\n",
+    "* `minimum_data` as the start time of the first biking period. This is also the first cutoff time for building features."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "lt = lm.search(\n",
+    "    df.sort_values('starttime'),\n",
+    "    num_examples_per_instance=-1,\n",
+    "    minimum_data='2014-06-30 08:00',\n",
+    "    verbose=False,\n",
+    ")\n",
+    "\n",
+    "lt.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The output from the search is a label times table with three columns:\n",
+    "\n",
+    "* The station ID associated to trips.\n",
+    "* The start time of the biking period. This is also the cutoff time for building features. Only data that existed beforehand is valid to use for predictions.\n",
+    "* The number of trips. This is calculated by our labeling function.\n",
+    "\n",
+    "As a helpul reference, we can print out the search settings that were used to generate these labels."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "lt.describe()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can also get a better look at the labels by plotting the distribution and cumulative count across time."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "fig, ax = subplots(nrows=2, ncols=1, figsize=(6, 8))\n",
+    "lt.plot.distribution(ax=ax[0])\n",
+    "lt.plot.count_by_time(ax=ax[1])\n",
+    "fig.tight_layout(pad=2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Feature Engineering\n",
+    "\n",
+    "In the previous step, we generated the labels. The next step is to generate the features.\n",
+    "\n",
+    "### Representing the Data\n",
+    "\n",
+    "We will represent the bike trips using an entity set. This way, we can generate features based on the relational structure of the dataset. We currently have a single table of trips where one station can have many trips. This one-to-many relationship can be represented by normalizing an entity for stations. The same can be done with other one-to-many relationships. Let's start by structuring the entity set."
```

Comment:
We've typically been using `entityset` as a single word in the Featuretools docs.

Reply:
Also might explain that you need to normalize the dataset because you want to make predictions based on the `from_station` so we need a station entity that can be used as the target entity for the feature generation process.